db/system_keyspace.cc

```diff
@@ -2021,13 +2023,15 @@ static bool maybe_write_in_user_memory(schema_ptr s, database& db) {
             || s == v3::scylla_views_builds_in_progress();
 }
 
-future<> make(database& db, service::storage_service& ss) {
-    register_virtual_tables(ss);
+}
```

Comment:
This closing brace makes this set is non-git-bisect-compilable. Overall the PR compiles because next patch removes the brace that's few lines above this.

Reply:
Oops, I make this bisectable but probably messed up later. I'll fix.

Reply:
I tested, it's bisectable. So it's probably some confusing diff artifact.

Reply:
What happens is: We add closing bract to a namespace, and replace make() with system_keyspace_make(). If the diff was written

```
+}
-future<> make(database& db, service::storage_service& ss) {
-     register_virtual_tables(ss);
+future<> system_keyspace_make(database& db, service::storage_service& ss) {
+     system_keyspace::register_virtual_tables(ss);
```

then it would be clear, but diff puts the deletions ahead of the additions.

Reply:
(I'm awarding myself two points for this)